README.md

```diff
@@ -378,215 +378,33 @@ Some standouts:
 
 ## Installation
 
-JAX is written in pure Python, but it depends on XLA, which needs to be
-installed as the `jaxlib` package. Use the following instructions to install a
-binary package with `pip` or `conda`, to use a
-[Docker container](#docker-containers-nvidia-gpu), or to [build JAX from
-source](https://jax.readthedocs.io/en/latest/developer.html#building-from-source).
+### Supported platforms
 
+|            | Linux x86_64 | Linux aarch64           | Mac x86_64   | Mac ARM        | Windows x86_64 | Windows WSL2 x86_64 |
+|------------|--------------|-------------------------|--------------|----------------|----------------|---------------------|
+| CPU        | yes          | yes (build from source) | yes          | yes            | yes            | yes                 |
+| NVIDIA GPU | yes          | yes (build from source) | no           | no             | no             | experimental        |
+| Google TPU | yes          | no                      | no           | no             | no             | no                  |
```

Comment:
Consider using a N/A in columns where the combination doesn't make sense - eg, I don't think there's any situation where one can run a TPU directly from a Mac ARM.